sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -290,12 +290,12 @@
      * </p>
      * @return cached signers.
      */
-    Mono<List<AttestationSigner>> getCachedAttestationSigners() {
+    Mono<List<AttestationSigner>> getCachedAttestationSigners(Context context) {
         if (this.cachedSigners.get() != null) {
             return Mono.just(this.cachedSigners.get());
         } else {
-            return this.signerImpl.getAsync()
-                .map(AttestationSignerImpl::attestationSignersFromJwks)
+            return this.signerImpl.getWithResponseAsync(context)
```

Comment:
To make sure I understand this we added context through all the Rest API calls for open telemetry to work correctly.

Reply:
Almost exactly correct. When I removed the sync stubs, the getAsync method was removed and I changed the code to use getWithResponseAsync. But I originally screwed up and didn't plumb context through, so while working through the telemetry changes, I fixed that.